examples/change_attribute/change_attitude.py

```diff
@@ -0,0 +1,160 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+"""
+© Copyright 2017, Team OpenMakerDrone at Yonsei Univ.
```

Comment:
This might be a problem with the license and the project rules. Maybe somebody else can weight in.